docker/domserver/scripts/start.d/50-domjudge.sh

```diff
@@ -87,6 +89,32 @@ else
 	echo "TRUSTED_PROXIES=${DOCKER_GATEWAY_IP}" >> webapp/.env.local
 fi
 
+# Add custom trusted proxies
+if [ -n "${TRUSTED_PROXIES}" ]; then
+	PRE_TRUSTED_PROXIES=$(grep "TRUSTED_PROXIES" webapp/.env.local | sed 's|TRUSTED_PROXIES=||g')
+	sed -i "s|TRUSTED_PROXIES=.*|TRUSTED_PROXIES=${PRE_TRUSTED_PROXIES},${TRUSTED_PROXIES}|" webapp/.env.local
+fi
+
+# Add trusted proxies for Nginx
```

Comment:
We don't need both. I'd say we always add it only to nginx, since that works with this approach (adding it to the end)